DQM/BeamMonitor/test/log4cplus.properties

```diff
@@ -0,0 +1,14 @@
+# Set root logger level to WARN and its only appender to A1.
+log4cplus.rootLogger=WARN, A1

```

Comment:
I am not sure if this characters at the end of the line are expected, they look like coming from emacs....

Reply:
It was a copy of the default from log4cplus, will be fixed.